projects/core/src/cms/store/actions/page.action.ts

```diff
@@ -1,48 +1,66 @@
-import { Action } from '@ngrx/store';
+import { PAGE_DATA_ENTITY } from '../cms-state';
+import {
+  EntitySuccessAction,
+  EntityLoadAction,
+  EntityFailAction
+} from '../../../state';
 import { Page } from '../../model/page.model';
 import { PageContext } from '../../../routing/index';
 
+export const LOAD_PAGE_INDEX = '[Cms] Load Page Index';
```

Comment:
LOAD_PAGE_INDEX is confusing. How about "UPDATE_PAGE_INDEX".

Reply:
Sounds fine with me. /cc @dunqan what's you opinion?

Reply:
What about just "LOAD_PAGE", ""LOAD_PAGE_SUCCESS", ""LOAD_PAGE_FAIL" and "UPDATE_PAGEDATA" ?

Reply:
so the related question is, do we need 'LOAD_PAGEDATA' and 'LOAD_PAGEDATA_FAIL'?

Reply:
I like "LOAD_PAGEDATA", "LOAD_PAGEDATA_FAIL" and "LOAD_PAGEDATA_SUCCESS". "LOAD_PAGE" sounds like download page content. Actually, we want to download cms page data. These 3 actions extends `entityLoader` actions, so they change the flags.
And, we also need 1 action, which can save the page data into `pageData` entities (when success ). How about call it "ADD_PAGEDATA"?

Reply:
:+1: for "LOAD_PAGEDATA", "LOAD_PAGEDATA_FAIL" and "LOAD_PAGEDATA_SUCCESS".
I'm thinking about removing separate actions for PAGE_DATA completely, but it would require some changes in reducers and LOAD_PAGEDATA_SUCCESS action.